test/test_data/apache_http_server.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+mapping_test_data = [
+    {
+        "product": "httpd",
+        "version": "2.4.51",
+        "version_strings": ["Apache/2.4.51"],
+    },
+]
+
+package_test_data = [
+    {
+        "url": "http://mirrors.kernel.org/fedora/releases/35/Everything/x86_64/os/Packages/h/",
+        "package_name": "httpd-2.4.51-2.fc35.rpm",
+        "product": "httpd",
```

Comment:
As above, this test will fail if there's a mismatch between httpd and http_server